### PR TITLE
Update settings.php

### PR DESCRIPTION
**Pull request by rakekniven**
Tiny l10n improvement.

Reported at Transifex: https://www.transifex.com/nextcloud/nextcloud/translate/#de_DE/user_ldap/91002448

### CODE REVIEW

File: apps/user_ldap/templates/settings.php [hunk 1]
```diff
@@ -121,7 +121,7 @@
...
+		<p class="ldapIndent"><?php p($l->t('Usernames are used to store and assign (meta)data. In order to precisely identify and recognize users, each LDAP user will have an internal username. This requires a mapping from username to LDAP user. The created username is mapped to the UUID of the LDAP user. Additionally the DN is cached as well to reduce LDAP interaction, but it is not used for identification. If the DN changes, the changes will be found. The internal username is used all over. Clearing the mappings will have leftovers everywhere. Clearing the mappings is not configuration sensitive, it affects all LDAP configurations! Never clear the mappings in a production environment, only in a testing or experimental stage.'));?></p>
```
Comment (blizzz) [inline apps/user_ldap/templates/settings.php]: The original way looks add, I agree. Perhaps an additional hyphen within the brackets, `(meta-)`, what do you think?

Comment (rakekniven) [reply apps/user_ldap/templates/settings.php]: Check some web resources and now I am asking why using brackets? Metadata would fit and work for me.
@comradekingu what do you think?

Comment (blizzz) [reply apps/user_ldap/templates/settings.php]: Also fine with that

Comment (MorrisJobke) [reply apps/user_ldap/templates/settings.php]: I'm in for dropping the brackets 👍

**Review by blizzz — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Pull request merged**